.travis.yml

```diff
@@ -1,18 +1,41 @@
 dist: xenial
 language: python
-python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
-  - "3.7"
+matrix:
```

Comment:
Do we need to test the same version of Ansible on different pythons? Should we be able to trust that Ansible itself has tested its code on different pythons and for us we only care that we work for certain pythons and certain Ansible versions?

Reply:
Ansible, not, but we occasionally run into differences between Python 2.7, 3.5 and 3.7 (3.6 so far always behaved like 3.7) around sorting of dicts, which at least once found an actual bug in my code.